app/assets/stylesheets/omni_nav/omni_nav_v1/modules/omni-nav.scss

```diff
@@ -40,8 +40,8 @@ $cu-viewport-xl: 1200px;
 }
 
 @mixin font-stack-for-headings() {
-    font-family: "futura-pt", Arial, Verdana, "Sans Serif";
-    font-family: "futura-pt", Arial, Verdana, "Sans Serif";
+    font-family: "futura-pt", Arial, "Sans Serif";
```

Comment:
I discussed it with Ross and we are not going to make any changes to the OmniNav (v1 or v2) in this story. So please remove all changes to the stylesheets under the `app/assets/stylesheets/omni_nav/` folder. Also remove changes to the files in the `build/omninav/` directory, generated by `rake build:omninav`.

Reply:
@kaylaziegler I can't actually remove the file changes generated by the build without starting over in a new branch, but I have made the updates and rebuilt, so they should essentially be the same code but with different timestamps.

Reply:
@sharissis-brei You actually can, it's just a matter of pulling the master versions of those files. I took care of it, so don't worry about it! Just make sure you don't run `rake build:omninav`

Reply:
@kaylaziegler Oh wow! I didn't know there was a way to do that via Git. I just copied over files manually for the omni nav stylesheets before I got to this comment. Would you mind taking a look at the latest commit and let me know if it's showing the changes you expect?